content/docs/fields/group.md

```diff
@@ -7,7 +7,36 @@ consumes:
     details: Shows group field interface and how to use
 ---
 
-The `group` field represents a group of values. This field is best used when there is a single group to be edited, typically with a single JSON object or nested frontmatter values. If there are multiple groups, checkout the [group-list](/docs/fields/group-list) field.
+The `group` field represents a group of values. This field is best used when there is a **single** group to be edited, typically with a single JSON object or nested frontmatter values. If there are multiple groups, checkout the [group-list](/docs/fields/group-list) field.
+
+![tinacms-date-field](/img/fields/group.gif)
+
+## Options
+
+```typescript
+import { Field } from '@tinacms/core'
+
+interface GroupConfig {
+  name: string
+  component: 'group'
+  label?: string
+  fields: Field[]
+}
+```
+
+| Option        | Description                                                                                                                                              |
+| ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`        | The path to some value in the data being edited.                                                                                                         |
+| `component`   | The name of the React component that should be used to edit this field. Available field component types a[defined here](/docs/fields)                    |
+| `label`       | A human readable label for the field. This label displays in the sidebar and is optional. If no label is provided, the sidebar will default to the name. |
+| `description` | An optional description of the field.                                                                                                                    |
+| `fields`      | An array of group values that will render as a sub-menu.                                                                                                 |
```

Comment:
I've been switching up the text a little bit. See the [text field docs](https://tinacms.org/docs/fields/text) for reference. The big things are

1. making `component` the first in the table
1. stating the `component` should always be "group"
1. putting `_(Optional)_` at the end of label and description
1. removing reference to the sidebar
1.

Reply:
Should be fixed now